scripts/ensure_min_python_is_tested.py

```diff
@@ -43,3 +43,21 @@
             file=sys.stderr,
         )
         sys.exit(1)
+
+
+proc = subprocess.run(
+    ["python", "-m", "mddj", "read", "tox", "min-version"],
```

Comment:
Can mddj simply be imported? It strikes me as odd to run mddj instead of calling some function in it collect a value via STDOUT.

Reply:
The issue here is that I've only defined/documented CLI interfaces for `mddj`.

I could add library/functional interfaces as well, but I haven't done so yet.

If I'm going to do this, it would be a dedicated `mddj.api` subpackage which declares itself as the public library interface. Does that sound sensible? `mddj.api.read.tox.min_version()` as the callable to use? (Some classes or other namespace type to make that path reflect the CLI)

I'll do this if you think it's worth it, but I wouldn't want to start trying to use any existing functions from there.

Reply:
Let's not force the interface of mddj just so this feedback has an actionable outcome. Let's just mark this as resolved.

Reply:
Cool by me! I'm happy to circle back on this later though. I like the idea of trying to build an explicit functional interface to match the CLI -- seems like a fun side-project when I have some time to give it a shot.